### PR TITLE
Added 2 websites to dark-sites.config

### PR DESCRIPTION
**Pull request by garamlee500**
Both websites are dark.

### CODE REVIEW

File: src/background/index.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <html>
-
```
Comment (Gusted) [inline src/background/index.html:2]: ?

File: src/background/index.html [hunk 2]
```diff
@@ -8,5 +7,4 @@
 <body>
     <script src="index.js"></script>
 </body>
-
```
Comment (Gusted) [inline src/background/index.html:11]: ?

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -91,6 +91,7 @@ cascadr.co
 cdnnow.pro
 cheat.sh
 checkra.in
+chess.com
```
Comment (Gusted) [inline src/config/dark-sites.config:94]: https://www.chess.com/register
My eyes, this isn't dark.

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -225,6 +226,7 @@ game-debate.com
 gamebanana.com
 gamejolt.com
 gameloop.fun
+garamlee500.github.io
```
Comment (Gusted) [inline src/config/dark-sites.config:229]: This is indeed dark.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dark-sites.config (hunk 3 above)
Comment (garamlee500) [reply src/config/dark-sites.config:94]: Can I exclude a sub website?

Comment (Gusted) [reply src/config/dark-sites.config:94]: Nope

Comment (garamlee500) [reply src/config/dark-sites.config:94]: Ok thanks, I'll remove chess.com then

File: src/background/index.html (hunk 1 above)
Comment (garamlee500) [reply src/background/index.html]: Undid commit

File: src/background/index.html (hunk 2 above)
Comment (garamlee500) [reply src/background/index.html]: Undid commit

**Pull request merged**